src/ruby/yast/ops.rb

```diff
@@ -66,11 +66,9 @@ module Ops
     # @!method                     self.get_locale(       obj, idx, def )
     #   @return [String, nil]   {Convert.to_locale}({get}(obj, idx, def))
     Ops::SHORTCUT_TYPES.each do |type|
-      eval <<END, binding, __FILE__, __LINE__ + 1
-        def self.get_#{type}(object, indexes, default=nil, &block)
-          Yast::Convert.to_#{type} get(object, indexes, default, 1, &block)
-        end
-END
+      define_singleton_method("get_#{type}") do |object, indexes, default=nil, &block|
```

Comment:
Does a default value for a block argument work?? This (old) post says it does not: https://www.ruby-forum.com/topic/121058 Do we have a test for this case?

Reply:
in new ruby it should, but I will add test.